odpf/entropy/v1beta1/service.proto

```diff
@@ -166,3 +172,19 @@ message GetLogRequest {
 message GetLogResponse {
   LogChunk chunk = 1;
 }
+
+message ResourceHistory {
+  string urn = 1;
+  map<string, string> labels = 2;
+  google.protobuf.Timestamp created_at = 3;
+
+  google.protobuf.Value difference = 4;
+}
+
+message GetResourceHistoryRequest {
+  string urn = 1;
```

Comment:
should we paginate?

Reply:
Left out pagination for now, we can do that on all APIs at once.

Reply:
Planned to add it right away, will do it for other APIs as part of this itself.

Reply:
Dropped pagination for now, need more details on the pagination UX.